canonical-web-and-design/ubuntu.com
# Changes to modal step 1

## Done

- Updated step 1 according to [the issue](https://github.com/canonical-web-and-design/commercial-squad/issues/167)
- Refactored the steps of the modal
- Removed any calls to update customer info anon as we don't display any preview related content in the first step anymore.
 

## QA

- go on [/advantage/subscribe?test_backend=true](https://ubuntu-com-10317.demos.haus/advantage/subscribe?test_backend=true)
- check that the first step matches the specs
- check that I didn't break anything


## Issue / Card

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/167

## Screenshots

![image](https://user-images.githubusercontent.com/11927929/132332376-19b160c4-6168-42a4-9152-eef5fb45f226.png)
